pkg/cmd/validate.go

```diff
@@ -83,26 +84,28 @@ func NewRootCommand() *cobra.Command {
 		RunE:  invoked.Run,
 	}
 	res.Flags().StringVarP(&invoked.version, "version", "", "", "Kubernetes version to validate native resources against. Required if not connected directly to cluster")
-	res.Flags().StringVarP(&invoked.localFilesDir, "local-schemas", "l", "", "--local-schemas=./path/to/schemas/dir. Path to a directory with format: /apis/<group>/<version>.json for each group-version's schema.")
+	res.Flags().StringVarP(&invoked.localSchemasDir, "local-schemas", "l", "", "--local-schemas=./path/to/schemas/dir. Path to a directory with format: /apis/<group>/<version>.json for each group-version's schema.")
+	res.Flags().StringVarP(&invoked.localCRDsDir, "local-crds", "", "", "--local-crds=./path/to/crds/dir. Path to a directory containing .yaml or .yml files for CRD definitions.")
 	res.Flags().StringVarP(&invoked.schemaPatchesDir, "schema-patches", "", "", "Path to a directory with format: /apis/<group>/<version>.json for each group-version's schema you wish to jsonpatch to the groupversion's final schema. Patches only apply if the schema exists")
 	res.Flags().VarP(&invoked.outputFormat, "output", "o", "Output format. Choice of: \"human\" or \"json\"")
 	clientcmd.BindOverrideFlags(&invoked.kubeConfigOverrides, res.Flags(), clientcmd.RecommendedConfigOverrideFlags("kube-"))
 	return res
 }
 
 func (c *commandFlags) Run(cmd *cobra.Command, args []string) error {
-	// Tool fetches openapi in the following priority order:
+	// tool fetches openapi in the following priority order:
```

Comment:
Did `gofmt` or something change the comments to lowercase?

Reply:
Nope, was just me :)
Some comments started with upper case letter and others with lower case...

Reply:
Do you want me to revert the changes ?

Reply:
nah was just curious